python/tests/petab_test_problems/lotka_volterra/model/convert_to_petab.sh

```diff
@@ -0,0 +1,21 @@
+petab_path=../petab
```

Comment:
```suggestion
#!/bin/bash
set -eou pipefail
script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
petab_path=$script_dir/../petab
```

+ add script_dir to other filenames used below